sklearn/tests/test_isotonic.py

```diff
@@ -535,3 +535,15 @@ def test_isotonic_thresholds(increasing):
         assert all(np.diff(y_thresholds) >= 0)
     else:
         assert all(np.diff(y_thresholds) <= 0)
+
+
+def test_isotonic_strict():
+    # check on enforcing strict monotonicity
+    X = np.array([0.1, 0.3, 0.5, 0.7, 1.])
+    y = np.array([1, 0, 1, 0, 1])
```

Comment:
I think we could test on more data, e.g. on the same kind of data that is used in the example https://scikit-learn.org/dev/auto_examples/miscellaneous/plot_isotonic_regression.html#sphx-glr-auto-examples-miscellaneous-plot-isotonic-regression-py

Reply:
Done !